### ISSUE TITLE
Saving lyrics doesn't work for audio files from network shares

### ISSUE THREAD

**Issue by eremem** [jacquesh/foo_openlyrics#159]
I access my audio file from a NAS via a Windows network share (eg. \\\\nas01\Multimedia\Audio\....).
The lyrics are shown in the panel but not being saved neither to the album's directory nor as tags.
If I choose some local directory, everything works as expected.

**Comment by craigo-** [comment]
I can confirm the exact same symptoms here. This occurs when the save directory is set to "Save to the same directory as the track" (NB: not the default setting).

When the save directory is set to "Save to the configuration directory" (the default), lyrics are successfully saved locally (in my case, to "%appdata%\foobar2000\lyrics").

To try and track down the problem I turned on "Debug logging enabled", but cannot locate where this log is saved. Can you tell me where it is?

Environment:
* foobar2000 1.6.11
* Windows 10 Enterprise 21H2
* foobar2000 library on NAS being accessed via SMB3 share, the user account has read/write permission

**Comment by craigo-** [comment]
Thank you. The relevant debug lines are as follows:

```
INFO-OpenLyrics: Saving lyrics to a local file...
INFO-OpenLyrics: Save file name format '[%tracknumber%] [%title%]' with directory class 'TrackFileDirectory' evaluated to 'file://\\homenas01\music\The Psychedelic Furs\Mirror Moves\01 The Ghost In You'
ERROR-OpenLyrics: Failed to save lyrics to Local files: Invalid path syntax
```

**Comment by jacquesh** [comment]
fyi this is fixed for v1.3 (the next release) but until that's released, the following workaround should get it working:

Map the network share as a network drive (so that Windows assigns it a drive letter) and then add the mapped drive path to your library (instead of the network share path, the one starting with "\\<machine-name>\...").
